src/Web/Slack/Types/Channel.hs

```diff
@@ -22,7 +23,7 @@ data Channel = Channel { _channelId         :: ChannelId
                        , _channelMembers    :: Maybe [UserId]
                        , _channelTopic      :: Maybe Topic
                        , _channelPurpose    :: Maybe Purpose
-                       , _channelIsMember   :: Bool
+                       , _channelIsMember   :: Maybe Bool
```

Comment:
No; `Nothing` means we don't know anything about it (the API didn't return this field).